business_requirement/tests/test_br.py

```diff
@@ -113,19 +113,21 @@ def test_create_sequance(self):
             'name': '/',
             'description': 'test',
         }
-        self.br.create(br_vals)
+        res = self.br.create(br_vals)
+        self.assertTrue(res.name)
```

Comment:
@YogeshMahera-SerpentCS 
I want you to compare a sentence lets say:
`br_name = "BR0001/test"`
Then 
`self.assertEqual(br_name, res.name)`